lesson_17/dags/sql/transfer_customers_data_lake_raw_to_bronze.sql

```diff
@@ -0,0 +1,42 @@
+DECLARE v_dag_logical_date DEFAULT CAST(CAST('{{ dag_run.logical_date }}' AS TIMESTAMP) AS DATE);
+DECLARE v_dag_start_date DEFAULT CAST('{{ dag_run.start_date }}' AS TIMESTAMP);
+DECLARE v_dag_ds DEFAULT CAST(CAST('{{ ds }}' AS TIMESTAMP) AS DATE);
+
+TRUNCATE TABLE `{{ params.project_id }}.bronze.customers`
+;
```

Comment:
If the data is not pretty big, I use a partitioned table in Bronze, but use not a partitioned table in Silver and MERGE data from Bronze to Silver. 
Keeping partitioned table in Bronze helps with debugging. Sometimes a lot.